intial changes to cloud starter guide (#948)
tinacms/tinacms.org: content/guides/tina-cloud/starter/structure.md

### Explore the GraphQL API

> If you have a GraphQL client like [Altair](https://altair.sirmuel.design/) go to `http://localhost:4001/graphql` to browse the docs and query our GraphQL API.

Reviewer: We now embed Altair by default.

```suggestion
To browse the docs and query our GraphQL API, run 'yarn dev` in your Tina project and go to `http://localhost:4001/altair`.
```
